### PR TITLE
implemented Horia feature

### PR DESCRIPTION
**Pull request by MateiRaduConstantin**
Implemented stuff

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
```
Comment (MateiRaduConstantin) [inline .idea/misc.xml:2]: Nu e bine deloc

Comment (MateiRaduConstantin) [reply .idea/misc.xml:2]: Ba da

File: src/HoriaFeature.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+public class HoriaFeature {
+    public static String bestJokeEver = "True";
```
Comment (oana27) [inline src/HoriaFeature.java:2]: Not funny

**Comment by MateiRaduConstantin** [comment]
am facut merge iei

**Pull request merged**